services/appConfig.js

```diff
@@ -0,0 +1,41 @@
+/* global define */
+
+(function (root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define([
+      'stratus',
+      'underscore',
+      'angular'
+    ], factory)
+  } else {
+    factory(root.Stratus, root._, root.angular)
+  }
+}(this, function (Stratus, _, angular) {
+  Stratus.Services.appConfig = [
+    '$provide', function ($provide) {
+      $provide.factory('appConfig', [
+        '$q', '$http', function ($q, $http) {
+          function facebookAppId () {
+            return '244333872792483'
+          }
+
+          function googleAppId () {
+            return '560074394524-csqefvbcv1mgjjqr65vs3skabk4m1vbc.apps.googleusercontent.com'
+          }
+
+          function s3Credentials () {
+            return {
+              POLICY: 'ewogICJleHBpcmF0aW9uIjogIjIwMjAtMDEtMDFUMDA6MDA6MDBaIiwKICAiY29uZGl0aW9ucyI6IFsKICAgIHsiYnVja2V0IjogInNpdGV0aGVvcnljb3JlIn0sCiAgICBbInN0YXJ0cy13aXRoIiwgIiRrZXkiLCAiIl0sCiAgICB7ImFjbCI6ICJwcml2YXRlIn0sCiAgICBbInN0YXJ0cy13aXRoIiwgIiRDb250ZW50LVR5cGUiLCAiIl0sCiAgICBbInN0YXJ0cy13aXRoIiwgIiRmaWxlbmFtZSIsICIiXSwKICAgIFsiY29udGVudC1sZW5ndGgtcmFuZ2UiLCAwLCA1MjQyODgwMDBdCiAgXQp9',
+              SIGNATURE: '5bz7ETqEC0Gxj1vJP/a6t3DRMJc=',
+              ACCESS_KEY: 'AKIAIX32Y3S7HCX4BSQA'
```

Comment:
Why are S3 Credentials posted here?

Reply:
At first, they were in uploadToS3 function (media.js). I just want to move everything config to the same file to easily manage. What do you think?

Reply:
I didn't realize they were there before.  This is unfortunate, as it exposes our Access Key to anyone who would have searched for it.  Anyway, we don't need the system to do any uploads to S3 directly, as everything goes through our Server-Side Microservice for processing and control.

Reply:
It looks like this was introduced in commit https://github.com/Sitetheory/stratus/commit/7bb50c432f43a817c4edda420adf9dd15b886760

Reply:
ok. I will update this commit. And I will remove the S3Credentials in another commit.